tools/perf/profiles/brave-typical-mac.zip.sha1

```diff
@@ -1 +1 @@
-46ac38accaee043e44799f655ab6fd21a3903c2f
+18a04aa2976235bf307dbae92add32c9b55f380a
```

Comment:
the URL to download: https://perf-data.s3.brave.com/perf-profiles/18a04aa2976235bf307dbae92add32c9b55f380a

This profiles doesn't have `ads_service` anymore: https://github.com/brave/brave-core/pull/27517